proto/inspect.proto

```diff
@@ -0,0 +1,120 @@
+syntax = "proto3";
+
+option go_package = ".;pb";
+
+// Distro denotes a product line of operating systems, using the following
+// test:
+//   If two operating systems at the same version and CPU architecture can be
+//   imported using the same logic, then they have the same Distro. For example,
+//   if Ubuntu 20.04 and xubuntu 20.04 are importable using
+//   the same logic, then they'd both be categorized as Distro.UBUNTU.
+//
+// When adding new members, keep in mind:
+//  - Group distros by family, using buckets of size 1000.
+//  - The following properties are orthogonal and should not be encoded here:
+//      - CPU architecture
+//      - Major or minor versions
+//      - GCE licensing (such as BYOL)
+enum Distro {
+  DISTRO_UNKNOWN = 0;
+
+  WINDOWS = 1000;
+
+  DEBIAN = 2000;
+  UBUNTU = 2001;
+  KALI = 2002;
+
+  OPENSUSE = 3000;
+  SLES = 3001;
+  SLES_SAP = 3002;
+
+  FEDORA = 4000;
+  RHEL = 4001;
+  CENTOS = 4002;
+  AMAZON = 4003;
+  ORACLE = 4004;
+}
+
+enum Architecture {
+  ARCHITECTURE_UNKNOWN = 0;
+  X86 = 1;
+  X64 = 2;
+}
+
+// OsRelease records the name and version of an operating system.
+message OsRelease {
+  // cli_formatted is a concatenation of distro, major_version, and
+  // minor_version using the format expected by the `--os` flag.
+  // For examples, see:
+  //  https://cloud.google.com/sdk/gcloud/reference/compute/images/import#--os
+  string cli_formatted = 1;
```

Comment:
Maybe it makes sense to refer to CLI as gcloud. Looking at this code base, one can assume CLI is actually the wrapper (which in a way it is, but we use it as backend in our services).

Reply:
That's a good point. The fields map to //google3/logs/proto/compute_image_tools/compute_image_tools_log.proto, and `cli_formatted` is used there, though.